crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -86,6 +88,12 @@ pub struct RealRibosome {
     //      - is already in the wasm cache, and only include the DnaDef portion
     //      - here in the ribosome.
     pub dna_file: DnaFile,
+
+    /// Entry and link types for each integrity zome.
+    pub zome_types: Arc<GlobalZomeTypes>,
```

Comment:
I have been a bit murky for a while on my understanding of what a Ribosome even *is* anymore, and now that more fields are being added, I'm not sure if it's getting murkier or clearer. I think that's OK though. "It is what it is" applies well here, or alternatively, "it is what it needs to be". ("It's not the ribosome that we deserve, but the one we need"?) Not a criticism of this PR at all, just some musing on the various directions Ribosome has taken over the years.

"Ribosome" might be a concept ripe for renaming too, but it's not urgent since this is entirely an internal concept that no DNA dev has to know about.